src/infrastructure/jira/jira-service.ts

```diff
@@ -29,9 +33,12 @@ export type AttachedDesignUrlV2IssuePropertyValue = {
 	readonly name: string;
 };
 
+export type IngestedDesignUrlIssuePropertyValue = string;
+
 export const propertyKeys = {
 	ATTACHED_DESIGN_URL: 'attached-design-url',
 	ATTACHED_DESIGN_URL_V2: 'attached-design-url-v2',
+	INGESTED_DESIGN_URLS: 'ingested-design-urls',
```

Comment:
**suggestion (non-blocking):** Taking into account that issue properties are global, consider adding an app-specifc prefix to minimize the risk of name collisions with other apps (e.g., `figma-for-jira:ingested-design-urls`).